download.dd

```diff
@@ -104,7 +104,8 @@ $(DIVC download_channels,
   )
 
   $(DIVC download_channel,
-    $(H2 $(LNAME2 dmd-nightly,DMD Nightly))
+    $(H2 $(LNAME2 dmd-nightly,DMD Nightly ($(NIGHTLIES_DMDV2))))
+    $(LINK2 changelog/$(NIGHTLIES_DMDV2)_pre.html, Changelog)
```

Comment:
If you think it's critical, feel free to go ahead. Otherwise I can get to it tomorrow.